app/controllers/pusher_controller.rb

```diff
@@ -4,10 +4,27 @@ class PusherController < ApplicationController
   # stop rails CSRF protection for this action
   protect_from_forgery except: :auth
 
+  # POST /pusher/auth
+  # @argument channel_name [String] The name of the pusher channel.
+  # @argument socket_id [String] The id of the pusher socket.
+  #
+  # Returns an auth token from Pusher that looks like this
+  # {"auth": ":7944d5b3dc2915f6a9c55dbedcbe91037799bb3627fc868890c2a7fd6f5fde4b"}
+  # 
+  # If there is an error, the response will look like this
+  # {"channel_name":["can't be blank"],"socket_id":["can't be blank"]}
```

Comment:
I think we should consider internationalization (i18n) here!

Reply:
Thanks for that suggestion!